contents/docs/libraries/next-js.md

```diff
@@ -77,18 +81,83 @@ export default function App({ Component, pageProps }) {
 }
 ```
 
+### App router
+
+If your Next.js app to uses the [app router](https://nextjs.org/docs/app), you can integrate PostHog by creating a `providers.js` file in your app folder. This is because the `posthog-js` library needs to be initialized on the client-side using the Next.js `'use client'` directive. 
```

Comment:
is it worth hyperlinking the `use client directive` to the NextJs docs? Im not familiar with what this is

Reply:
Yup, done